server/sonar-web/src/main/js/app/components/nav/global/GlobalNav.js

```diff
@@ -30,13 +30,15 @@ import Tooltip from '../../../../components/controls/Tooltip';
 import HelpIcon from '../../../../components/icons-components/HelpIcon';
 import OnboardingModal from '../../../../apps/tutorials/onboarding/OnboardingModal';
 import { getCurrentUser, getAppState, getGlobalSettingValue } from '../../../../store/rootReducer';
+import { skipOnboarding } from '../../../../store/users/actions';
```

Comment:
It's not really related to this ticket (maybe a later one that I didn't see yet), but while testing onboarding I noticed that if I go to the last step, then delete the project I just created with the tutorial, and finally I click the skip tutorial button : then I end up on a dashboard page that doesn't exist (for the project I created and the deleted).